dvc/commands/experiments/init.py

```diff
@@ -67,32 +73,63 @@ def run(self):
             interactive=self.args.interactive,
             force=self.args.force,
         )
+        self._post_init_display(initialized_stage, initialized_deps)
+        if self.args.run:
+            self.repo.experiments.run(targets=[initialized_stage.addressing])
+        return 0
 
-        text = ui.rich_text.assemble(
-            "Created ",
-            (self.args.name, "bright_blue"),
+    def _post_init_display(
```

Comment:
All of the outputs have been moved out of `dvc.experiments.init` now (except what we need for prompts) to `commands`.